Makefile.debug

```diff
@@ -106,7 +106,7 @@ control: control.o modes.o bandplan.o decode_status.o libradio.a
 cwd: cwd.o morse.o libradio.a
 	$(CC) $(LDOPTS) -o $@ $^ -lbsd -lpthread -lm
 
-rtlsdr: rtlsdr.o libradio.a
+rtlsdr: rtlsdrd.o libradio.a
```

Comment:
As reported by CI, this change broke compilation on Linux because this rule was not renamed to `rtlsdrd`.

Reply:
I've fixed it in #34.